src/main/utils/scanUtils.ts

```diff
@@ -27,4 +27,13 @@ export class ScanUtils {
     public static getHomePath(): string {
         return path.join(os.homedir(), '.jfrog-vscode-extension');
     }
+
+    /**
+     * Get scan exclude pattern. This pattern will exclude paths from scanning.
```

Comment:
Get scan exclude pattern. This pattern will exclude paths from scanning. -->
Get scan exclude pattern. This pattern is used to exclude specific file descriptors (go.mod, package.json, etc.) from being scanned by Xray. Descriptor files which are under a directory which matches the pattern will not be scanned.